File: fos-api/FOS/FOS.Model/Domain/Event.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FOS.Model.Domain
{
    public class Event
    {
        public string Name { get; set; }
        public string Restaurant { get; set; }
        public string RestaurantId { get; set; }
        public string Category { get; set; }
        public string Participants { get; set; }
        public string MaximumBudget { get; set; }
        public string DeliveryId { get; set; }
        public string HostName { get; set; }
        public string HostId { get; set; }
        public string CreatedBy { get; set; }
        public DateTime? CloseTime { get; set; }
        public string EventId { get; set; }
        public string Status { get; set; }
        public DateTime? RemindTime { get; set; }
        public Boolean IsMyEvent { get; set; }
        public string EventType { get; set; }
        public DateTime? EventDate { get; set; }
        public string EventParticipantsJson { get; set; }
        public EventAction Action { get; set; }
        public string ServiceId { get; set; }
    }
}

File: fos-client/src/app/models/event.ts


    // $Classes/Enums/Interfaces(filter)[template][separator]
    // filter (optional): Matches the name or full name of the current item. * = match any, wrap in [] to match attributes or prefix with : to match interfaces or base classes.
    // template: The template to repeat for each matched item
    // separator (optional): A separator template that is placed between all templates e.g. $Properties[public $name: $Type][, ]

    // More info: http://frhagn.github.io/Typewriter/

    
    import { EventAction } from './event-action';

    export class Event    {
           
        public Name: string = null;   
        public Restaurant: string = null;   
        public RestaurantId: string = null;   
        public Category: string = null;   
        public Participants: string = null;   
        public MaximumBudget: string = null;   
        public DeliveryId: string = null;   
        public HostName: string = null;   
        public HostId: string = null;   
        public CreatedBy: string = null;   
        public CloseTime: Date = null;   
        public EventId: string = null;   
        public Status: string = null;   
        public RemindTime: Date = null;   
        public IsMyEvent: boolean = false;   
        public EventType: string = null;   
        public EventDate: Date = null;   
        public EventParticipantsJson: string = null;   
        public Action: EventAction = null;
    }
         
